corehq/apps/programs/templates/programs/manage/program.html

```diff
@@ -25,31 +26,36 @@
     {% endif %}
 {% endblock %}
 
-{% block main_column %}
+{% block page_content %}
     <form class="form form-horizontal" name="program" method="post">
-        {% bootstrap_form_errors form %}
-        {% bootstrap_fieldset form "Program Information" %}
-        <div class="form-actions"><button type="submit" class="btn btn-primary">
-            {% if program.get_id %}
-                {% trans 'Update Program' %}
-            {% else %}
-                {% trans 'Create Program' %}
-            {% endif %}</button></div>
+        <legend>{% trans "Program Information" %}</legend>
+        {% crispy form %}
+        <div class="form-actions">
+            <div class="col-sm-12">
```

Comment:
this should have the class `col-sm-4 col-md-5 col-lg-3 col-sm-offset-3 col-md-offset-4 col-lg-offset-2`